src/wallOverlap.cpp

```diff
@@ -202,15 +192,25 @@ int64_t WallOverlapComputation::getApproxOverlapArea(const Point from, const Poi
         const int64_t other_from_proj = dot(other_from - from, vec) / vSize(vec);
 
         const int64_t overlap_length_2 = from_proj + other_from_proj; //Twice the length of the overlap area, alongside the lines.
+        const int64_t overlap_width_2 = line_width * 2 - std::sqrt(LinearAlg2D::getDist2FromLineSegment(from, other_from, to)) - std::sqrt(LinearAlg2D::getDist2FromLineSegment(other_from, from, other_to));
```

Comment:
Same as above: The distance should already be stored in the link.

Reply:
Same as above reply.